src/prelude.rs

```diff
@@ -51,7 +51,7 @@ pub use auto::traits::ToolItemExt;
 pub use auto::traits::ToolShellExt;
 pub use auto::traits::TreeModelExt;
 //pub use auto::traits::TreeSortableExt;
-//pub use auto::traits::WidgetExt;
+pub use auto::traits::WidgetExt as WidgetExtGenerated;
```

Comment:
Ah, not quite what I was hoping for. If both traits are exported here, we can go back to just reexporting `auto::traits::*` and only reexport manual traits manually. So it would the manual one that would get renamed (I guess `WidgetExtManual`) but in its module rather than here.